src/user/service.ts

```diff
@@ -39,7 +40,7 @@ export async function findById(id: number): Promise<User> {
 }
 export async function findByIdentity(identity: string): Promise<User> {
```

Comment:
With this function and `findAll()` above, it's generally best to pass the entire user object as a parameter since you can guarantee the typeset. I can show you examples in the media-api another day. 

Doesn't have to change for this PR, but something to keep in mind going forward.